### DISCUSSION TITLE
Ajuda sobre modelgem de sistemas

### DISCUSSION THREAD

**Discussion by DronnDev** [frontendbr/forum#2157] [Perguntas e Respostas]
Boa noite devs, tudo certo? Sou iniciante no mundo da programaçao, sou estudante tecnico de desenvolvimento de sistemas. Estou iniciando o projeto do meu TCC no dia de hoje, e decidi programar um aplicativo. Ainda nao tenho grande conhecimento na area, estudei apenas modelagem de sistemas, e agora estou vendo sobre banco de dados. Meu professor pediu para que concluissemos a modelagem de nossos sistema, e me surgiram duvidas logo no começo. Voces tem alguma dica para facilitar o levantamento das regras de negocio, e dos requisitos funcionais e nao funcionais? Estou um pouco de dificuldade para fazer isso, nao sei se tem tanta diferença para a modelagem de dados de um app para de um sistema empresarial, por exemplo. Se puderem me ajudar, fico grato desde ja.

   Muito obrigado!!!

**Comment by nazarepiedady** [comment]
Revise o conteúdo da máteria UML
